fullscreen.bs

```diff
@@ -505,9 +518,15 @@ properties apply to this pseudo-element either.
 <p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
 <a>element</a> that has its <a>fullscreen flag</a> set.
 
-<p class="note no-backref">This makes it different from the {{Document/fullscreenElement}} API,
-which returns a <a>document</a>'s <a>fullscreen element</a>.
+<p class="note no-backref">This makes it different from the fullscreenElement API, which returns
+returns the topmost fullscreen element..
```

Comment:
You droppped the reference to fullscreenElement here. And the double dot looks wrong.